### DISCUSSION TITLE
RVC on Kaggle

### DISCUSSION THREAD

**Discussion by bennett-krubakar** [w-okada/voice-changer#799] [Ideas]
If we can run RVC on google colab, we can also run it on other jupyter workspace such as Kaggle or Amazon Sagemaker Studio. Can we get a notebook to run it on Kaggle or Amazon Sagemaker as they give you more GPU runtime than google colab.

**Comment by ghost** [comment]
Yeah, would be great.

**Comment by Mkk-m** [comment]
Would be super. Is there any way to edit the code cos if I'm correct they are still jupyter editors

**Comment by Mkk-m** [comment]
especially for the new beatrice v2 trainer

**Comment by Mkk-m** [comment]
[notebook40ce84bc5d.zip](https://github.com/user-attachments/files/17617059/notebook40ce84bc5d.zip)
 
 
 turns out it can be done. here's a training notebook for kaggle 
 don't know if its well organize first time coding or whatever
